core/assembly/deserializer.ts

```diff
@@ -109,7 +109,7 @@ export abstract class Deserializer<I> {
     let value: T
 
     // @ts-ignore
-    if (value instanceof u128) { return this.decode_u128(); }  // -> we need to get ride of this
```

Comment:
Should we add a test in which this breaks?

Reply:
Yeah if you can think of one!